CHANGES.rst

```diff
@@ -1652,6 +1652,10 @@ Bug Fixes
 
   - Fixed bug in Ci definition. [#5106]
 
+  - Non-ascii cds unit strings are now correctly represented using ``str`` also
+    on python2. This solves bugs in parsing coordinates involving strings too.
+    [#5355]
```

Comment:
I thought about this too, and even had added and removed it, but in the end felt it was slightly more logical to refer to PRs only (and the links are there for the curious).